### PR TITLE
#31 Fixed the gramma of the .md files through Grammar.ly

### LINKED ISSUE #31: Run our *.md documents in this repo through the Grammer.ly service and update
#### Issue Summary

We would like to get our English polished up throughout the repo.

#### Aceptance Criteria

* Every .md file in this repo has been run through the Grammer.ly service and updated accordingly

### PR DESCRIPTION
**Pull request by ssiddhantsharma**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:

**Every .md file in this repo has been run through the Grammer.ly service and updated accordingly**

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=32) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=32) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/dx-automator?pullRequest=32) it.</sub>

File: services/client-react/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ OR
 ./scripts/setup-local-db
 ```
 
-If bash doesn't recognize the **REACT_APP_TASKS_SERVICE_URL** environment variable just export it and run the last two commands again.
+If the bash doesn't recognize the **REACT_APP_TASKS_SERVICE_URL** environment variable, just export it and run the last two commands again.
```
Comment (thinkingserious) [inline services/client-react/README.md]: "the " is not needed here, please delete.

File: services/github/README.md [hunk 2]
```diff
@@ -89,13 +89,13 @@ docker-compose -f docker-compose-prod.yml run github-service python manage.py te
 
 # Reference
 
-### Kill All Running Docker Instances
+### Kill all the Running Docker Instances
```
Comment (thinkingserious) [inline services/github/README.md]: All should still be capitalized.

File: services/github/README.md [hunk 3]
```diff
@@ -89,13 +89,13 @@ docker-compose -f docker-compose-prod.yml run github-service python manage.py te
...
 
 ```bash
 docker stop $(docker ps -a -q)
 ```
 
-### Check Environment on Server
+### Check Environment on the Server
```
Comment (thinkingserious) [inline services/github/README.md]: Check the Environment on the Server

File: services/looker/README.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
 
 SendsByLibrary / InvoicingByLibrary
-- inherits schema from DXLooker, with appropriate table name
+- Inherits the schema from DXLooker, with appropriate table name
 
 LookerAPIHandler  
-- handles authentication of Looker API
-- uses credentials to pull latest look data from Looker 
+- Handles the authentication of theLooker API
+- Uses the credentials to pull the latest look data fromthe  Looker
```
Comment (thinkingserious) [inline services/looker/README.md]: theLooker API should be the Looker API

fromthe should just be from

File: services/looker/README.md [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
+- Handles the authentication of theLooker API
+- Uses the credentials to pull the latest look data fromthe  Looker 
 
 DBCache
-- queries database to store all rows locally for faster access
+- Queries database to store all the rows locally for faster access
 
 Look
-- Stores Looker look id and LookerApiHandler instance
+- Stores Looker look the id and LookerApiHandler instance
```
Comment (thinkingserious) [inline services/looker/README.md]: Please remove "the "

File: README.md [hunk 6]
```diff
@@ -1,22 +1,22 @@
 # The Developer Experience (DX) Automator
 
-This tool is intended to help make managing multiple Github repositories much easier for DX, DevRel, and Open Source Engineering teams. 
+This tool is intended to help make managing, the multiple Github repositories much easier for DX, DevRel, and Open Source Engineering teams.
```
Comment (thinkingserious) [inline README.md]: Please do not edit this sentence. Thank you.

Comment (ssiddhantsharma) [reply README.md]: Thank you for reviewing! Resolved the issue.

File: services/client-react/README.md (hunk 1 above)
Comment (ssiddhantsharma) [reply services/client-react/README.md]: Thank you for reviewing! Resolved the issue.

File: services/github/README.md (hunk 2 above)
Comment (ssiddhantsharma) [reply services/github/README.md]: Thank you for reviewing! Resolved the issue.

File: services/client-react/README.md (hunk 1 above)
Comment (ssiddhantsharma) [reply services/client-react/README.md]: Thank you for reviewing! Resolved the issue.

File: services/github/README.md (hunk 3 above)
Comment (ssiddhantsharma) [reply services/github/README.md]: Thank you for reviewing! Resolved the issue.

**Comment by thinkingserious** [comment]
Hello @ssiddhantsharma,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**